x-pack/plugins/security/public/authentication/login/login_page.tsx

```diff
@@ -83,14 +89,22 @@ const loginFormMessages: Record<LogoutReason, NonNullable<LoginFormProps['messag
 };
 
 export class LoginPage extends Component<Props, State> {
-  state = { loginState: null } as State;
+  state = { loginState: null, customBranding: {} } as State;
+  private customBrandingSubscription?: Subscription;
 
   public async componentDidMount() {
     const loadingCount$ = new BehaviorSubject(1);
+    this.customBrandingSubscription = this.props.customBranding.customBranding$.subscribe(
```

Comment:
Cool I get why you would use this.props.customBranding.customBranding$ here since we're subscribing to the observable